src/styles.css

```diff
@@ -1,8 +1,91 @@
+.notion {
```

Comment:
I tried to keep the naming of the CSS variables consistent, but Notion's not super consistent internally.